src/routers/controllers/loginController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+import db from '../../models';
+import UserService from '../../services/usersService';
+import { controllerErrorHandler } from '../../utils';
+
+const UserModel = db.user;
+
+const userServiceInstance = new UserService(UserModel);
+
+export const loginController = async ({ body: { userName, password }, params: { id } }: Request, res: Response) => {
+    try {
+        const user = await userServiceInstance.readUser(id, true);
+
+        if (!user) return res.status(404).json({ message: `User with id ${id} not found` });
```

Comment:
Auth-related logic should not show the DB state. It would be treated as vulnerability. If the user does not exist, you should respond with something like 401: "Your username or password doesn't match"